### PR TITLE
Data source PagerDuty Integration

### PR DESCRIPTION
**Pull request by mazzy89**
This PR aims to add `signalfx_pagerduty_integration` data source

### CODE REVIEW

File: signalfx/data_source_pagerduty_integration.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func dataSourcePagerDutyIntegrationRead(d *schema.ResourceData, meta interface{}) error {
+	return nil
```
Comment (mazzy89) [inline signalfx/data_source_pagerduty_integration.go]: The lookup here would be done by Integration Name. However the Go SDK does not have this function implemented yet.

Need to create it in the Go SDK and then port it.

Comment (mazzy89) [reply signalfx/data_source_pagerduty_integration.go]: This has been ported

**Comment by mazzy89** [comment]
@keitwb this PR follows the PR you have approved on Friday on signalfx Go SDK.

**Review by jrcamp — APPROVED** [review]
Thanks for adding!

**Comment by mazzy89** [comment]
Thanks a lot @jrcamp. do you plan to wire a release any time soon?

**Comment by jrcamp** [comment]
@mazzy89 do you need a release made?

**Comment by jrcamp** [comment]
I'll make one. 👍

**Comment by mazzy89** [comment]
oh great @jrcamp. again thanks a lot for such level of proactiveness. really appreciated.

**Comment by jrcamp** [comment]
@mazzy89 just realized, I think the data source has to be registered here: https://github.com/splunk-terraform/terraform-provider-signalfx/blob/7bfe97183397fd744b2a02ffc9a58b5aff49151a/signalfx/provider.go#L65 Can you  make sure it works before I spin the release?

**Comment by mazzy89** [comment]
oh that's true. @jrcamp let me test it then and i will ping you here

**Pull request merged**